docs/src/safe_mode.md

```diff
@@ -0,0 +1,35 @@
+# Safe Mode
```

Comment:
I don't think this is a good idea -- it's currently called safe mode everywhere else. I think we should keep the naming consistent within a version, rather than have two quite different things in different places.